portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -24,6 +24,7 @@
 import javax.servlet.http.HttpServletResponse;
 import com.google.gson.Gson;
 
+/** Servlet that checks if a user is logged in or not. */
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
```

Comment:
this comment is for the if statement in line 37

You can write this concisely as:
`LoginStatus status = new LoginStatus(userService.isUserLoggedIn(), userService.createLogoutURL("/comments.htlm"));`

Reply:
This doesn't seem to work for me because it doesn't let me log in. If the user is not logged in, it should create a login url instead so I think I have to use createLoginURL.

Reply:
Oh good point, I missed that there are different methods for login and logout